test.sh

```diff
@@ -1,7 +1,23 @@
 #!/bin/bash
-echo "Checking gofmt..."
-files=$(gofmt -l -s ./tests)
-if [[ $files ]]; then
-    echo "Gofmt errors in files: $files"
-    exit 1
+set -e
+set -x
+export TAG=$2
+export REPO=$1
+export VERSION=$3
+export BUCKET=$4
+GCLOUD_CMD="gcloud"
+if [ -n "$5" ]
+then
+  GCLOUD_CMD=$5
 fi
+
+if [ "$VERSION" == "jessie" ]
```

Comment:
just for my own understanding: why take version as a parameter at all if we only support building jessie? are we planning on supporting wheezy (or other versions) in the future?